### ISSUE TITLE
Support BPMN2 sub-processes

### ISSUE THREAD

**Issue by adamfowleruk** [adamfowleruk/scxmltocpf#7]
Allow a process flow to initiate and return from a sub process flow.
Be aware of escalation timers, and parallel execution use cases too alongside this feature

**Comment by adamfowleruk** [comment]
Depends upon #4

**Comment by adamfowleruk** [comment]
escalation timers postponed until sprint-004, so out of scope for this enhancement request

**Comment by eouthwaite** [comment]
moved to sprint-004 following Adam's note above
